openshift.go

```diff
@@ -1,3 +1,4 @@
+//go:build openshift
```

Comment:
@tylerauerbeck is this change intentional ?

Reply:
@iam-veeramalla Fixed.